graalpy-install-package/src/main/java/com/oracle/example/graalpy/Main.java

```diff
@@ -62,40 +63,77 @@ public static void main(String[] args) throws Exception {
+  private static final String HOME_PREFIX = "/vfs/home";
+  private static final String PROJ_PREFIX = "/vfs/proj";
+
+  public static Context getContext() {
+    VirtualFileSystem vfs = VirtualFileSystem.newBuilder()
+      .extractFilter(p -> {
+        String s = p.toString();
+        // Specify what files in the virtual filesystem need to be accessed outside the Truffle sandbox.
+        // e.g. if they need to be accessed by the operating system loader.
+        return s.endsWith(".ttf");
+      })
+      .build();
+    Context context = Context.newBuilder()
+      // set true to allow experimental options
+      .allowExperimentalOptions(false)
+      // setting false will deny all privileges unless configured below
+      .allowAllAccess(false)
+      // allows python to access the java language
+      .allowHostAccess(true)
+      // allow access to the virtual and the host filesystem, as well as sockets
+      .allowIO(IOAccess.newBuilder()
+              .allowHostSocketAccess(true)
+              .fileSystem(vfs)
+              .build())
+      // allow creating python threads
+      .allowCreateThread(true)
+      // allow running Python native extensions
+      .allowNativeAccess(true)
+      // allow exporting Python values to polyglot bindings and accessing Java from Python
+      .allowPolyglotAccess(PolyglotAccess.ALL)
+      // choose the backend for the POSIX module
+      .option("python.PosixModuleBackend", "java")
+      // equivalent to the Python -B flag
+      .option("python.DontWriteBytecodeFlag", "true")
+      // equivalent to the Python -v flag
+      .option("python.VerboseFlag", System.getenv("PYTHONVERBOSE") != null ? "true" : "false")
+      // log level
+      .option("log.python.level", System.getenv("PYTHONVERBOSE") != null ? "FINE" : "SEVERE")
+      // equivalent to setting the PYTHONWARNINGS environment variable
+      .option("python.WarnOptions", System.getenv("PYTHONWARNINGS") == null ? "" : System.getenv("PYTHONWARNINGS"))
+      // print Python exceptions directly
+      .option("python.AlwaysRunExcepthook", "true")
+      // Force to automatically import site.py module, to make Python packages available
+      .option("python.ForceImportSite", "true")
+      // The sys.executable path, a virtual path that is used by the interpreter to discover packages
+      .option("python.Executable", vfs.resourcePathToPlatformPath(VENV_PREFIX) + (VirtualFileSystem.isWindows() ? "\\Scripts$\\python.exe" : "/bin/python"))
+      // Set the python home to be read from the embedded resources
+      .option("python.PythonHome", vfs.resourcePathToPlatformPath(HOME_PREFIX))
+      // Do not warn if running without JIT. This can be desirable for short running scripts
+      // to reduce memory footprint.
+      .option("engine.WarnInterpreterOnly", "false")
+      // Set python path to point to sources stored in src/main/resources/vfs/proj
+      .option("python.PythonPath", vfs.resourcePathToPlatformPath(PROJ_PREFIX))
```

Comment:
I'm not sure all of these options are really needed, or whether you just want to document them. Maybe you could add `(optional)` or so to the end of the comments for options that aren't really needed?